lib/dl_connector_ydb/dl_connector_ydb_tests/db/core/test_adapter.py

```diff
@@ -0,0 +1,21 @@
+from dl_core_testing.testcases.adapter import BaseAsyncAdapterTestClass
+from dl_testing.regulated_test import RegulatedTestParams
+
+from dl_connector_ydb.core.ydb.adapter import YDBAdapter
+from dl_connector_ydb.core.ydb.target_dto import YDBConnTargetDTO
+from dl_connector_ydb_tests.db.core.base import BaseYDBTestClass
+
+
+class TestAsyncYDBAdapter(
+    BaseYDBTestClass,
+    BaseAsyncAdapterTestClass[YDBConnTargetDTO],
+):
+    test_params = RegulatedTestParams(
+        mark_tests_failed={
+            BaseAsyncAdapterTestClass.test_pass_db_query_to_user: "Error transformer not implemented",  # TODO: FIXME
+            BaseAsyncAdapterTestClass.test_default_pass_db_query_to_user: "Error transformer not implemented",  # TODO: FIXME
+            BaseAsyncAdapterTestClass.test_timeout: "Error transformer not implemented",  # TODO: FIXME
```

Comment:
If these fails are actually related to the lack of error transformer, we can mark them with an existing ticket BI-5885